SUMMARY.md

```diff
@@ -85,6 +85,7 @@
 - [API](apps/api/README.md)
 - [Application System](apps/application-system/README.md)
 - [Financial Aid System for Samband Sveitarfélaga](apps/financial-aid/README.md)
+  - [Financial Aid Web Shared](apps/financial-aid/web-shared/README.md)
```

Comment:
For consistency, this should be a library under libs. App projects should not import stuff from other app projects. 

If this contains stuff that should only be shared between financial-aid (web) projects you can move it to for example `libs/financial-aid/components`.

Reply:
Very true, fixed, will was at it then I also changed the alias for other files in the financial-aid system.